### ISSUE TITLE
Bug: SerializedTableRowNode type has height as required

### ISSUE THREAD

**Issue by BarnabyShearer** [facebook/lexical#4873]
Lexical version: v0.11.3

## Steps To Reproduce

```
  const row: SerializedTableRowNode = {
    type: table
    children: [],
    version: 1,
    direction: "ltr",
    format: "left",
    indent: 0,
    height: undefined,
  };
```

## The current behavior

`TableRowNode` has `__height?: number`  and everything works fine with an undefined height (with the row height based on its contents). 

However we get a type error: `Type 'undefined' is not assignable to type 'number'.ts(2322)`

## The expected behavior

No type error.

**Comment by NextThread** [comment]
can I work on this

**Comment by CailBzd** [comment]
Hello, Have you made progress on the subject to date?

**Comment by ivailop7** [comment]
Was fixed in #5696
